root/release/index.tt

```diff
@@ -6,7 +6,7 @@
 
     [% IF release.mediums.size == 0 %]
       <p>
-        [% l('The tracklist for this release is currently unknown.') %]
+        [% l('We have no information about this release’s media and tracklist.') %]
```

Comment:
This was reworded to "We have no information about ..."  while the new medium specific string still uses the previous wording ("The tracklist for this medium is unknown"). Should we make those consistent?

Reply:
I reworded it to make it more general, so I'm not quite sure how to make it consistent ("The media for this release are unknown" sounds off to me). But open to suggestions :)